src/components/pages/ParentHome/ChildrenList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Card, Avatar } from 'antd';
+import 'antd/dist/antd.css';
+import {
+  SettingOutlined,
+  EditOutlined,
+  EllipsisOutlined,
+} from '@ant-design/icons';
+
+function ChildrenList() {
+  const { Meta } = Card;
+
+  return (
+    <Card
```

Comment:
This is solid work for now. Later on when we apply functionality, we can easily modify this to map over a list of students.

Reply:
Definitely! We're just waiting on backend so we get a good idea of how we should build things out